src/settings.js

```diff
@@ -18,18 +18,48 @@ export default {
   },
   2018: {
     name: 'ISIC 2018 Challenge',
-    taskIds: [
-      45,
-      46,
-      47,
+    tasks: [
+      {
+        id: 45,
+        type: 'segmentation',
```

Comment:
The leaderboard details are probably broken for Task 1 and 2 now, but may be fixable.